packages/ir-sdk/fern/apis/ir-types-latest/definition/http.yml

```diff
@@ -58,6 +58,7 @@ types:
       requestBody: optional<HttpRequestBody>
       sdkRequest: optional<SdkRequest>
       response: optional<HttpResponse>
+      v2Responses: optional<list<HttpResponse>>
```

Comment:
can we make this an object that contains the list of responses so that its easy to evolve over time